docs/pages/deploy-a-cluster/helm-deployments.mdx

```diff
@@ -9,8 +9,11 @@ layout: tocless-doc
 These guides show you how to set up a full self-hosted Teleport deployment using
 our `teleport-cluster` Helm chart.
 
+- [Teleport Cluster](./helm-deployments/kubernetes-cluster.mdx): Running a Teleport cluster in a Kubernetes cluster using
```

Comment:
This removes SEO un-friendly "ing" and also provides a subject for the sentence.

```suggestion
- [Teleport Cluster](./helm-deployments/kubernetes-cluster.mdx): Run a Teleport cluster in a Kubernetes cluster using
```